scripts/trivy_vulnerabilities_data.sh

```
@@ -0,0 +1,218 @@
+      continue
+    fi
+
+    # Check if the entry already exists
+    if [[ -n "${existing_entries["$product,$scanner_tool,$vurn_id"]}" ]]; then
+      echo "Entry for $vurn_id already exists in the database. Skipping."
+      continue
+    fi
+
+    local pr_id="$GITHUB_PR_ID"
+    local pr_link="$GITHUB_PR_LINK"
+    local created_date=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)